backup.c

```diff
@@ -1245,13 +1246,17 @@ pg_stop_backup(pgBackup *backup)
 	res = execute("SET client_min_messages = warning;", 0, NULL);
 	PQclear(res);
 
-	params[0] = "false";
-	res = execute("SELECT * FROM pg_stop_backup($1)", 1, params);
+	/*
+	 * This parameter default is true,pg_backup_stop will wait for WAL
+	 * pg_backup_startto be archived when archiving is enabled.
```

Comment:
Do we really need to the description of default value?

Why don't you change the following?
* wait the last WAL segment that is required to form a complete set of backup files is archived.

Or is it ok to remove this comment because the comment of pg_backup_stop() says same things(line 1212)?

Reply:
No, We don't need to description this value. So I delete this comment.